scripts/dev

```diff
@@ -1,5 +1,10 @@
 #!/bin/bash
 
+set -e
+cd $(dirname $0)/..
+
+./scripts/postinstall
```

Comment:
The issue with that is you have to remember to synchronise the package.json files; but I suppose you could write a test that enforces that so 👍